Resolve ESLint warnings in ContributorThumbnail.js

### Overview
This file has ESLint errors or warnings:

```
src/components/ContributorThumbnail.js
  52:13  warning  Unexpected console statement  no-console
```

### Action Items
Resolve the errors or warnings. If there is a good reason to keep any piece of offending code, disable ESLint as shown below.

### Resources/Instructions
To see all ESLint errors and warnings, run this from the command line: `npm run lint`
Or for only one file, run this: `npx eslint path/to/file.js`

To close this issue by disabling errors or warnings when there is a reason that they cannot be resolved, use comments. 
To disable a rule for the whole file, put this at the top of the file:
`/* eslint-disable rule-name */`
To disable a rule for one line, put this right before the line:
`// eslint-disable-next-line rule-name`
Or put this at the end of the line:
`// eslint-disable-line rule-name`
Additional commenting options can be found in the [ESLint docs](https://eslint.org/docs/user-guide/configuring#disabling-rules-with-inline-comments).

### Dependency
This issue is one of many new issues this morning intended to replace the multi-file issue #169 with one issue per file.

--- a/src/components/ContributorThumbnail.js
+++ b/src/components/ContributorThumbnail.js
@@ -49,7 +49,9 @@ const Thumbnail = ({ thumbnailInfo, organization }) => {
           src={thumbnailInfo.imageUrl}
           className={classes.thumbnailImage}
           onError={(e) =>
+            // eslint-disable-next-line no-console
             console.log(`${e}: error with ${organization.name} image`)
+            // Before MVP: Refactor as on-website error/generic case.
           }
           alt={`${organization.name} logo`}
           loading="lazy"

--- a/src/components/ParentContributor.js
+++ b/src/components/ParentContributor.js
@@ -19,10 +19,12 @@ export const ParentContributor = ({ dropdownLength, children }) => {
               src={"https://codeforall.org/assets/images/homepage/logo.png"}
               className={classes.codeForAllThumbnailImage}
               onError={(e) =>
+                // eslint-disable-next-line no-console
                 console.log(
                   e,
                   `error with 'Code For All' ParentContributor Component`
                 )
+                // Before MVP: Refactor as on-website error message / generic case.
               }
               alt="code for all logo"
               loading="lazy"

